### PR TITLE
Guess Money : Simple math game in python

### PR DESCRIPTION
**Pull request by SiddhantTotade**
### Describe your change:
I have added a guess_money.py - a simple game in math. You have to think of any amount and it will guess how much money left in your pocket.

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [x] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms include at least one URL that points to Wikipedia or another similar explanation.
* [x] If this pull request resolves one or more open issues then the description above includes the issue number(s) with a [closing keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue): "Fixes #ISSUE-NUMBER".

### CODE REVIEW

File: other/guess_money.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+"""
+1. Do not enter any amount that you are thinking.
+2. The player just have to press enter and that's it.
+3. If you are weak in simple calculations then take a calculator and then play.
+"""
+
+
+class Guess:
+    def calculate_number(self) -> int:
+        # Generate a random number between 1 and 1000
```
Comment (cclauss) [inline other/guess_money.py:31]: Let's add a test...
```suggestion
        """
        Generate a random number between 1 and 10,000

        >>> 1 <= Guess().calculate_number() <= 10_000
        True
        """
```

**Comment by cclauss** [comment]
This is a game, not an algorithm.  See CONTRIBUTING.md

**Comment by SiddhantTotade** [comment]
> This is a game, not an algorithm. See CONTRIBUTING.md

Sir, but this is a simple algorithm which has a game perspective. If it is a game then there must be a condition of win or lose. An algorithm calculates all of the cases but a game let you win or lose. but this is an algorithm which perform calculations on the amount and return it. So this is not a game but a small algorithm that's why I created this after reading CONTRIBUTING.md.

**Comment by SiddhantTotade** [comment]
> This is a game, not an algorithm. See CONTRIBUTING.md

Also sir if you look at the `guess_the_number_search.py` in the same directory. It has a game perspective but it is an algorithm.

**Comment by tianyizheng02** [comment]
I'm siding with @cclauss on this. In order to have any meaningful distinction between an "algorithm" and "any Python program", this is the definition outlined in our [contributing guidelines](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md#what-is-an-algorithm):
> An Algorithm is one or more functions (or classes) that:
> - take one or more inputs,
> - perform some internal calculations or data manipulations,
> - return one or more outputs,
> - have minimal side effects (Ex. print(), plot(), read(), write()).
> 
> Algorithms should be packaged in a way that would make it easy for readers to put them into larger programs.

In short, an algorithm is a function that performs calculations on a given input to produce an output in order to accomplish some general task. Think of the type of algorithms you learn in a computer science class: Dijkstra's algorithm, quick sort, depth-first search, extended Euclidean algorithm, etc. Going by this definition, your program is at the very least a **script** and not an algorithm by this definition. Furthermore, you yourself described your program as a "game" multiple times.
> Guess Money is a simple fun math game.

**Pull request closed without merging**